tox.ini

```diff
@@ -41,7 +41,7 @@ commands =
     python setup.py check -m -r -s -v
 
 deps =
-    docutils
+    docutils==0.12
```

Comment:
`docutils` _just_ released `0.13.1` which caused our builds to fail inexplicably (see, e.g., [this failing build](https://travis-ci.org/dropbox/stone/jobs/182914200) vs. [this passing one](https://travis-ci.org/dropbox/stone/jobs/182914480), where the only difference is that the latter pins `docutils` to `0.12`). I'm pretty sure this is a bug in `docutils` (it appears on Travis CI in both Python 2.7.9 and 3.4.2, but not on my local machine with Python 2.7.10 and Python 3.5.2). I will follow up with the maintainers of that project. However, until a fix is released, this should remain pinned.